VideoCallKitQuickStart/ViewController+CallKit.swift

```diff
@@ -80,7 +92,8 @@ extension ViewController : CXProviderDelegate {
     func provider(_ provider: CXProvider, perform action: CXEndCallAction) {
         NSLog("provider:performEndCallAction:")
 
-        TVIAudioController.shared().stopAudio()
+        // AudioDevice is enabled by default
+        self.audioDevice.isEnabled = true;
```

Comment:
I think we want it disabled by default.

Reply:
We can. I was thinking we will change it back to the default value after callkit use is done. WDYT? I thought, during the review, @idelgado was suggesting it to reset to TVIDefaultAudioDevice's property `enabled`'s default value to  `true`.

I can go either way on this.